ui/raidboss/data/02-arr/trial/titan-ex.ts

```diff
@@ -67,13 +70,17 @@ export default {
       id: 'TitanEx Rock Throw',
       netRegex: NetRegexes.tether({ id: '0007' }),
       suppressSeconds: 1,
-      alertText: (data, matches, output) => {
-        if (matches.source === data.me || matches.target === data.me)
-          return output.jailOnYou();
+      // TODO: this is pretty darn verbose, so maybe we need Matches<TetherParams>
+      // or possibly even just pre-define them all into MatchesTether or something.
+      alertText: (data: Data, matches: Matches<NetRegex<TetherParams>>,
+          output: Output): string | undefined => {
+        if (matches?.source === data.me || matches?.target === data.me)
+          return output.jailOnYou?.();
       },
-      infoText: (data, matches, output) => {
-        if (matches.source !== data.me && matches.target !== data.me)
-          return output.jails();
+      infoText: (data: Data, matches: Matches<NetRegex<TetherParams>>,
```

Comment:
One interesting thing is that this function (only) made me specify the output type of the function and also the types of Data/Matches/Output.  However, other functions (above) did not.

Reply:
@Trim21 Do you understand why I have to specify types here but not on other similar functions?

Reply:
Ok, fixed this by removing the `@typescript-eslint/explicit-module-boundary-types` rule for raidboss triggers, which I think makes sense.